src/pool/srv.c

```diff
@@ -72,11 +76,107 @@ fini(void)
 	return 0;
 }
 
+static int
+recreate_pool_layout()
```

Comment:
The function name looks bit confusing to me, maybe recreate_pool_files() is a better name.

Reply:
changed the name to recreate_pooltgts()